app/screens/thread/thread.js

```diff
@@ -26,6 +26,18 @@ export default class Thread extends PureComponent {
 
     state = {};
 
+    shouldComponentUpdate(nextProps) {
```

Comment:
This seems unnecessary since those other things changing should cause a rerender, and I can't see them changing very often.

Reply:
Oh trust me, they change whenever store changes, say a new post in another channel/team

Reply:
Ah right, since the post list isn't memoized properly